Ryujinx.HLE/HOS/Horizon.cs

```diff
@@ -335,7 +356,8 @@ public void ReadControlData(Nca controlNca)
         {
             IFileSystem controlFs = controlNca.OpenFileSystem(NcaSectionType.Data, FsIntegrityCheckLevel);
 
-            IFile controlFile = controlFs.OpenFile("/control.nacp", OpenMode.Read);
+            Result rc = controlFs.OpenFile(out IFile controlFile, "/control.nacp", OpenMode.Read);
```

Comment:
```suggestion
            Result resultCode = controlFs.OpenFile(out IFile controlFile, "/control.nacp", OpenMode.Read);
```
and add an empty line after.